README.md

```diff
@@ -83,51 +92,97 @@ interface-group <interface-group>
+8c07adf421fb        cyberreboot/poseidon:master            "/bin/sh -c '(flask …"   Up 2 hours (healthy) 
+0a4f947f299b        cyberreboot/vent-file-drop:master      "/bin/sh -c '(flask …"   Up 2 hours (healthy) 
+511f90c6ddd3        cyberreboot/crviz:master               "serve -s build -l 5…"   Up 2 hours (healthy)   
+fb250044ff17        cyberreboot/poseidon-api:master        "/bin/sh -c '(flask …"   Up 2 hours (healthy)  
+8e898fd68c08        cyberreboot/vent-network-tap:master    "/bin/sh -c '(flask …"   Up 2 hours (healthy)   
+552f65d7a982        cyberreboot/vent-rq-worker:master      "/bin/sh -c '(flask …"   Up 2 hours (healthy)  
+8dbabe78d1b9        cyberreboot/vent-rq-worker:master      "/bin/sh -c '(flask …"   Up 2 hours (healthy)  
+e076452c1515        cyberreboot/vent-rq-worker:master      "/bin/sh -c '(flask …"   Up 2 hours (healthy)   
+d0f406f240b1        cyberreboot/vent-rq-worker:master      "/bin/sh -c '(flask …"   Up 2 hours (healthy)    
+6229e46723a9        cyberreboot/vent-rq-dashboard:master   "/bin/sh -c '(flask …"   Up 2 hours (healthy)   
+5c695040603b        cyberreboot/vent-redis:master          "docker-entrypoint.s…"   Up 2 hours (healthy)  
+004e5fdde96e        cyberreboot/vent-syslog:master         "/usr/sbin/syslog-ng…"   Up 2 hours (healthy)   
+c0cd7c1f881c        cyberreboot/vent-rabbitmq:master       "docker-entrypoint.s…"   Up 2 hours (healthy)   
+d81f5509628c        cyberreboot/vent                       "/bin/sh -c '(flask …"   Up 2 hours (healthy) 
+```
+
+If you performed the demo installation, you should also see the following Faucet-related containers running:
+
+```
+196f53632485        grafana/grafana:5.2.1                  "/run.sh"                Up 2 hours             
+cfb7d68b66e0        prom/prometheus:v2.3.1                 "/bin/prometheus --c…"   Up 2 hours             
+86f4188d67f6        faucet/gauge:latest                    "/usr/local/bin/entr…"   Up 2 hours            
+2e186573532e        faucet/event-adapter-rabbitmq          "/usr/local/bin/entr…"   Up 2 hours             
+c0652a6ccd44        influxdb:1.6-alpine                    "/entrypoint.sh infl…"   Up 2 hours             
+2a949b5b1687        faucet/faucet:latest                   "/usr/local/bin/entr…"   Up 2 hours             
+```
 
-After installation you'll have a new command `poseidon` available for looking at the status, logs, changing the configuration or stopping and starting the service.
+2. You should see "Poseidon successfully started, capturing logs..." in your syslog output:
 ```
-poseidon help
+neo@Poseidon01:/opt/poseidon/docs# journalctl -u poseidon | grep capturing
+Jul 25 15:42:20 greg-Poseidon01 poseidon[4273]: Poseidon successfully started, capturing logs...
 ```
 
-Results of Poseidon are printed out in the logs.
+To continue to test (assuming demo installation), please see '/opt/poseidon/docs/demo.txt'
```

Comment:
please change '/opt/poseidon/docs/demo.txt' to `/opt/poseidon/docs/demo.txt` (backticks apply the markdown)